content/docs/azure/deploy/install-kubeflow.md

```diff
@@ -92,39 +93,38 @@ Run the following commands to set up and deploy Kubeflow. The code below include
 
     # Generate and deploy the app:
     cd ${KFAPP}
     kfctl generate all -V
-    kfctl apply k8s -V
+    kfctl apply all -V
     ```
 
-    * ${KFAPP} - the name of a directory where you want Kubeflow configurations to be stored. This directory is created when you runkfctl init. If you want a custom deployment name, specify that name here. The value of this variable becomes the name of your deployment. The value of this variable cannot be greater than 25 characters. It must contain just the directory name, not the full path to the directory. The content of this directory is described in the next section.
+    * ${KFAPP} - the name of a directory where you want Kubeflow configurations to be stored. This directory is created when you run kfctl init. If you want a custom deployment name, specify that name here. The value of this variable becomes the name of your deployment. The value of this variable cannot be greater than 25 characters. It must contain just the directory name, not the full path to the directory.
 
 1. Check the resources deployed correctly in namespace `kubeflow`
 
         kubectl get all -n kubeflow
 
 1. Open Kubeflow Dashboard
 
-    ```
-    # You may choose to use a load balancer:
+The default installation does not create an external endpoint but you can use port-forwarding to visit your cluster. Run the following command and visit http://localhost:8080.
 
-    kubectl get svc –n kubeflow
-    kubectl expose svc ambassador -n kubeflow --type LoadBalancer --name <SVC_NAME>
-    kfctl apply k8s -V
+    kubectl port-forward svc/istio-ingressgateway -n istio-system 8080:80
 
-    # Find the external IP to access the dashboard:
-    kubectl get svc –n kubeflow
-    ```
+In case you want to expose the Kubeflow Dashboard over an external IP, you can change the type of the ingress gateway. To do that, you can edit the service:
+
+        kubectl edit -n istio-system svc/istio-ingressgateway
+
+From that file, replace `type: NodePort` with `type: LoadBalancer` and save.
 
-In this case, the external IP for the service named 'amb1' was 40.XX.XXX.XXX, so it was accessible at that address.
+While the change is being applied, you can watch the service until below command prints a value under the `EXTERNAL-IP` column:
 
-If you didn’t create a load balancer, please use port-forwarding to visit your cluster. Run the following command and visit localhost:8080.
+        kubectl get -w -n istio-system svc/istio-ingressgateway
 
-    kubectl port-forward svc/ambassador -n kubeflow 8080:80
+The external IP should be accessible by visiting http://<EXTERNAL-IP>. Note that above installation instructions do not create any protection for the external endpoint so it will be accessible to anyone without any authentication. You can read more about authentication from [Access Control for Azure Deployment](/docs/azure/authentication).
```

Comment:
Suggestion: At the end of the doc, link to this doc for further info about config with kfctl and kustomize:
https://www.kubeflow.org/docs/other-guides/kustomize/

Use a relative link - something like this:

```
You can find general information about Kubeflow configuration in the guide to [configuring Kubeflow with kfctl and kustomize](/docs/other-guides/kustomize/).
```